lib/systems/examples.nix

```diff
@@ -0,0 +1,130 @@
+# These can be passed to nixpkgs as either the `localSystem` or
```

Comment:
The same of this file is a bit odd, given that to actually use most of the platforms in `platforms.nix` you need to indirectly cross compile the bootstrap tools using this file, making them not examples anymore. Maybe `predefined-crosses` or something.

Though I guess in the longer run these should be merged into `platforms.nix` anyway...